v2/common/src/test/java/com/google/cloud/teleport/v2/transforms/AvroConvertersTest.java

```diff
@@ -111,19 +111,7 @@ public void testWriteWithoutSchema() {
     expectedException.expect(IllegalArgumentException.class);
     expectedException.expectMessage("withSchema(schema) called with null input.");
 
-    pipeline
-        .apply(
-            AvroConverters.ReadAvroFile.newBuilder()
-                .withInputFileSpec(AVRO_FILE_PATH)
-                .withSchema(SCHEMA_FILE_PATH)
-                .build())
-        .apply(
-            AvroConverters.WriteAvroFile.newBuilder()
-                .withOutputFile(FAKE_DIR)
-                .withSchema(null)
-                .build());
-
-    pipeline.run();
+    AvroConverters.WriteAvroFile.newBuilder().withOutputFile(FAKE_DIR).withSchema(null).build();
```

Comment:
I see no substantial difference between this test and the one below, but one calls `pipeline.run();` and the other one doesn't - why?

Reply:
@an2x fair point. Those tests don't break, but they never get to `pipeline.run();`, either, as the builder throws an exception. I have removed it to make the code consistent/clearer.
The problem is when there is actually something that succeeded to `pipeline.apply()`, but `pipeline.run()` isn't called -- which is the case for `ExpectedException` with multiple PTransforms.